### ISSUE TITLE
wallet cli: Remove manually created help text, use clap's text instead. 

### ISSUE THREAD

**Issue by sakeridge** [labs-space/solana#4]
Look at: https://github.com/solana-labs/solana/pull/885/files#diff-1d86fd2f81f93521de34cc67fa7dde6724289af1a18dbbbb64d88978832ce0ee

Should these all be eprintln!?

**Comment by mvilnes** [comment]
I don't have much of an opinion right now, we aren't super consistent about this. The previous help went to stdout and this patch preserves that behavior. Happy to add an e, lmk

**Comment by CrieasofCarrots** [comment]
Yeah, I think traditionally you'd only write usage() to stdout if --help.
